src/app/payment/components/checkout/SelectSeatsComponent.tsx

```diff
@@ -1,66 +1,99 @@
+import { useState } from 'react';
+import Button from '../../../shared/components/Button/Button';
+import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
+
 interface SelectUsersComponentProps {
   disableMinusButton: boolean;
   disablePlusButton: boolean;
+  maxSeats?: number;
+  minSeats?: number;
   seats: number;
-  onUsersChange: (seats: number) => void;
+  onSeatsChange: (seats: number) => void;
 }
 
 const SeparatorVertical = () => <div className="h-max border-[0.5px] border-gray-10 py-1" />;
 
 export const SelectSeatsComponent = ({
   disableMinusButton,
   disablePlusButton,
+  maxSeats,
+  minSeats,
   seats,
-  onUsersChange,
+  onSeatsChange,
 }: SelectUsersComponentProps): JSX.Element => {
+  const { translate } = useTranslationContext();
+  const [totalUsers, setTotalUsers] = useState<number>(seats);
+  const [showButton, setShowButton] = useState<boolean>(false);
+
+  const onInputChanges = () => {
```

Comment:
it seems that the name of the function does not reflect the behaviour of the function